build/common.project.props

```diff
@@ -169,6 +168,11 @@
     </AssemblyAttributes>
   </ItemGroup>
 
+  <!-- Assembly attributes for net core projects -->
+  <PropertyGroup Condition=" '$(IsNetCoreProject)' == 'true' ">
+    <AssemblyVersion>$(SemanticVersion).$(PreReleaseVersion)</AssemblyVersion>
```

Comment:
I created a simple netstandard library
```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFrameworks>netstandard1.3;net45</TargetFrameworks>
    <Version>2.3.4-beta1-1256</Version>
    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
  </PropertyGroup>

</Project>
```
And it did set version correctly. What's wrong with our projects?